include/paths.h

```diff
@@ -37,11 +37,20 @@
 
 #include <sys/cdefs.h>
 
+#ifdef COMPAT_64BIT
+#define	_PATH_LOCALBASE	"/usr/local64"
+#else
 #define	_PATH_LOCALBASE	"/usr/local"
+#endif
 
 /* Default search path. */
 #define	_PATH_DEFPATH	"/sbin:/bin:/usr/sbin:/usr/bin:" \
-  			_PATH_LOCALBASE "/sbin:" _PATH_LOCALBASE "/bin"
+			_PATH_LOCALBASE "/sbin:" _PATH_LOCALBASE "/bin" \
+#ifdef COMPAT_64BIT
```

Comment:
This still gives weird behaviour for two reasons:
1. freebsd64 binaries will prefer freebsd64 binaries over native ones
2. native binaries won't find freebsd64 ones
I think this should be unconditional, with native preferred over freebsd64?

Reply:
I suspect we'll need to change other places as well but that's a sensible default for CheriBSD and simpler. I'll just remove the use of _PATH_LOCALBASE entirely in _PATH_DEFINE.